nUpdate.Shared/Updating/UpdateResult.cs

```diff
@@ -47,6 +47,44 @@ internal class UpdateResult
                         config.Architecture == Architecture.X64 && !is64Bit)
                         continue;
 
+
+
+                    if (config.RolloutConditions != null && config.RolloutConditions.Count != 0)
+                        if (conditions == null || !conditions.Any())
+                        {
+                            continue;
+                        }
+                        else
+                        {
+                            if (config.RolloutConditions.Any(cond => conditions.Select(s => s.Key).Contains(cond.Key)))
+
+                            {
+                                bool doUpdate = false;
+                                foreach (var localCondition in conditions)
+                                {
+                                    doUpdate = config.RolloutConditions.Where(n => !n.IsNegativeCondition).Any(c =>
+                                        c.Key == localCondition.Key &&
+                                        String.Equals(c.Value, localCondition.Value, StringComparison.CurrentCultureIgnoreCase));
```

Comment:
It's generally recommended to use `string` instead of `String` any time.

Reply:
Thanks, i generally write ever in VB, this is my first time i write in C#. I will automatic you uppercase ;-)